src/common-principles.md

```diff
@@ -267,41 +274,38 @@ A guide for using macros can be found at
 -->
 {{ MACROS___make_filetree_example(
     {
-    "my_dataset-1": {
-            "sourcedata": "",
-            "...": "",
-            "rawdata": {
-                "dataset_description.json": "",
-                "participants.tsv": "",
+    "my_project-1": {
+        "sourcedata": {
+            "dicoms": {},
+            "raw": {
                 "sub-01": {},
                 "sub-02": {},
                 "...": "",
+                "dataset_description.json": "",
+				"...": "",
             },
-            "derivatives": {
-                "pipeline_1": {},
-                "pipeline_2": {},
-                "...": "",
-            },
+            "..." : "",
+        },
+        "derivatives": {
+            "pipeline_1": {},
+            "pipeline_2": {},
+            "...": "",
         }
     }
+   }
 ) }}
 
-In this example, where `sourcedata` and `derivatives` are not nested inside
-`rawdata`, **only the `rawdata` subdirectory** needs to be a BIDS-compliant
-dataset.
+In this example, `sourcedata/dicoms` is not nested inside
+`sourcedata/raw`, **and only the `sourcedata/raw` subdirectory** is a BIDS-compliant dataset among `sourcedata/` subfolders.
 The subdirectories of `derivatives` MAY be BIDS-compliant derivatives datasets
 (see [Non-compliant derivatives](#non-compliant-derivatives) for further discussion).
-This specification does not prescribe anything about the contents of `sourcedata`
-directories in the above example - nor does it prescribe the `sourcedata`,
-`derivatives`, or `rawdata` directory names.
-The above example is just a convention that can be useful for organizing raw,
-source, and derived data while maintaining BIDS compliance of the raw data
-directory. When using this convention it is RECOMMENDED to set the `SourceDatasets`
+The above example is just a convention useful for organizing source, raw BIDS, and derived BIDS data while maintaining BIDS compliance of the raw data directory.
```

Comment:
Couldn't this line could equally apply to the original structure? If this is enough to relieve the confusion, why not just do this and 

```diff
-In this example, where `sourcedata` and `derivatives` are not nested inside
-`rawdata`, **only the `rawdata` subdirectory** needs to be a BIDS-compliant
-dataset.
+In this example, `sourcedata` is not nested inside `rawdata`,
+**and only the `rawdata` subdirectory** is a BIDS-compliant dataset.
```

above?

Sorry to be really belaboring this, but I feel like a lot of effort is going into a minor clarification, and someone who notices this change is going to have to spend effort to understand that it's purely superficial.

Reply:
Yes, correct **but** my main problem with the original formulation is that we were suggesting something  which is **not** BIDS-compliant if adopted within a BIDS dataset, and context is not entirely clear here that we are not talking about a BIDS dataset. And if it is such a great idea to have `rawdata/` then why BIDS itself doesn't have it?

I agreed to remove `dataset_description.json` (which IMHO would have immediately made it "legit" according to spec, not validator, BIDS dataset), but I would like to preserve this adjusted recommended structure since if adopted within a BIDS dataset, it would remain legit, thus we are not somewhat misguiding a reader suggesting that `rawdata/` subdirectory is a legit way to describe BIDS-compliant dataset *within* a BIDS dataset.